src/crimson/os/seastore/cache.cc

```diff
@@ -764,6 +765,9 @@ record_t Cache::prepare_record(Transaction &t)
   record.extents.reserve(t.fresh_block_list.size());
   for (auto &i: t.fresh_block_list) {
     DEBUGT("fresh block {}", t, *i);
+    if (!i->is_relative()) {
```

Comment:
Let's make this is_inline().

Reply:
Wait, can the fresh_block_list contain ool extents?

Reply:
Yes, see https://github.com/ceph/ceph/pull/41803/files/9e0332c61fe61519bfafa584d3298fb9ea5c8c07#diff-73deebd5bf1b0d430c96dea6ca824d2bf630696addcc4ccc437d5b718d99218bR97-R103

Reply:
I think we need the `delayed_alloc_list` to hold only delayed ool extents, so that we can write them to disk before submitting the record

Reply:
In that case, add an is_inline() helper to use instead of is_relative() here.  I'd like it to be very obvious to readers which extents are being skipped.

Reply:
ok:-)